sdk/build.gradle.kts

```diff
@@ -0,0 +1,56 @@
+plugins {
+    id("com.hedera.gradlebuild.sdk.java")
+}
+
+dependencies.constraints {
+    // Define version of protobuf by dependency coordinates as both
+    // have the module name: 'com.google.protobuf'
+    val protobufVersion = "3.24.3"
+    versions("com.google.protobuf:protobuf-java:$protobufVersion")
+    versions("com.google.protobuf:protobuf-javalite:$protobufVersion")
+}
+
+moduleInfo {
+    version("com.github.spotbugs.annotations", "4.7.3")
+    version("com.google.common", "33.0.0-jre")
+    version("com.google.errorprone.annotations", "2.21.1")
+    version("com.google.gson", "2.10.1")
+    version("grpc.protobuf.lite", "1.46.0")
+    version("grpc.protobuf", "1.46.0")
+    version("headlong", "10.0.0")
+    version("io.grpc", "3.2.1")
+    version("io.grpc.stub", "1.58.0")
+    version("java.annotation", "1.3.2")
+    version("org.bouncycastle.pkix", "1.76")
+    version("org.bouncycastle.provider", "1.76")
+    version("org.slf4j", "2.0.9")
+    version("org.slf4j.simple", "2.0.9")
+
+    // Versions only required for tests and examples
+    version("java.dotenv", "5.3.1")
+    version("json.snapshot", "1.0.17")
+    version("kotlin.stdlib", "1.9.22")
+    version("org.assertj.core", "3.24.2")
+    version("org.apache.commons.lang3", "3.14.0")
+    version("grpc.netty.shaded", "1.57.2")
+}
+
+testModuleInfo {
+    requires("org.assertj.core")
+    requires("org.junit.jupiter.api")
+    requires("org.junit.jupiter.params")
+    requires("org.mockito")
```

Comment:
For tests I only added the minimum:
- `jupiter` does not need a version, because Gradle uses one of the latest as default
- `json.snapshot` brings in versions for "mockito"

More details: https://scans.gradle.com/s/awnnigj7pabpu/dependencies?configurationFilter=WyJ0ZXN0UnVudGltZUNsYXNzcGF0aCJd&expandAll&focusedDependencyView=versions&projectFilter=WyI6c2RrIl0&toggled=W1swLDAsWzE2NV1dLFswLDAsWzE2NSw0NjhdXV0

(You can run `./gradlew sdk:dependencies --scan` to get such a build scan with all the dependencies)

If you prefer, we can add versions for these dependencies as well.

Reply:
I am good with this setup for now, thanks 🙂